packages/core/test-utils/src/utils.js

```diff
@@ -42,7 +42,6 @@ export async function ncp(source: FilePath, destination: FilePath) {
           .on('error', reject);
       });
     } else if (stats.isDirectory()) {
-      outputFS.mkdirp(destPath);
```

Comment:
This wasn't awaited, and it also happens anyway in the next recursive call to `ncp`.